src/common/ant.ts

```diff
@@ -80,13 +80,20 @@ export class ANT {
    * Spawn a new ANT.
    */
   static spawn = spawnANT;
-  // no signer give read
-  static init(config: ANTConfigNoSigner): AoANTRead;
+  /**
+   * Fork an ANT to a new process.
+   *
+   * @param config
+   */
+  static fork = forkANT;
```

Comment:
i also wonder if we should add this on the processId as a AoWrite which would just call `fork` with itself

Reply:
Added